src/sections/About/index.js

```diff
@@ -34,7 +34,7 @@ const About = () => {
                             {" "}
                             <h4>About us</h4>
                             <h2>
-                                <span>Starting with Layer5</span> is easier than anything.
+                                <span>Starting with Layer5</span> is easier than before.
```

Comment:
What if I am a new contributor? Please revert this

Reply:
This area has yet to be updated from the original template, so we're essentially ok with anything here. We haven't done a first draft, yet.